app/scripts/controllers/push-platform-notifications/firebase/lib/firebase-app.js

```diff
@@ -0,0 +1,2735 @@
+/* eslint-disable */
```

Comment:
@matteoscurati could you include a link to where you copied the code from in this code comment?

Reply:
You're right. I'll include the license and the source of these libraries 👍

Reply:
Can we not use the npm package for this?

Reply:
Great question. Unfortunately, Lavamoat node doesn't support ESM, and Firebase does not expose an entry point in CJS. For this reason, at least for now, we had to import Firebase in the way indicated. If I understand correctly, this problem will be solved with the release of Endomoat

Reply:
Can we transpile Firebase using Babel instead? 

There is an example of how we did this in the past here: https://github.com/MetaMask/metamask-extension/pull/17251/files#diff-a05d1f077953d24c0d8ddecf64163da2653ba3b728e1ba1f18af6d770767b127R831-R841

Reply:
Ok, thanks to @FrederikBolding 's suggestion, I'm able to import Firebase using npm instead of importing the libraries directly!